getting-started-guide/event-data.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Events in Axiom"
+description: "This page explains the fundamentals of event data in Axiom."
+sidebarTitle: Events
+---
+
+In Axiom, event data is the atomic unit of activity, encompassing logs, traces, and even product interactions. Each event is simply a structured record—composed of key-value pairs—that captures meaningful interactions or changes in state within a system. While these can appear in various forms, from log lines to complex traces, they usually contain the following:
+
+- **Timestamp**: When the event occurred.
+- **Attributes**: A set of key-value pairs offering details about the event context.
+- **Metadata**: Contextual labels and IDs, such as trace ID and span ID, that connect related events.
+
+Rather than simply calling them logs or metrics, event data reflects a broader range of interactions, crossing boundaries from engineering to product management, security, and beyond. This includes everything from the number of documents created in a product to the sequence of API calls during a checkout process.
+
+## Types of event data
```

Comment:
I know that you want to set up the punchline of Axiom’s metrics compatibility, but the problem here is we have now gone from what I hope will be a world view that timestamped event data shows up everywhere, to narrowing down even _more_ than engineering to observability.

If I look at this page, I would definitely leave thinking Axiom is just another engineering-focused tool. How could we change that perception? 

Does it need a heading that shows we are narrowing to the engineering use case specifically? Is this section or mention of observability needed at all?